sev fails due to:
```
cp config-libkrunfw-sev_aarch64 linux-6.6.52/.config
cp: cannot stat 'config-libkrunfw-sev_aarch64': No such file or directory
make: *** [Makefile:77: linux-6.6.52] Error 1
```

--- a/pkgs/development/libraries/libkrunfw/default.nix
+++ b/pkgs/development/libraries/libkrunfw/default.nix
@@ -63,6 +63,6 @@ stdenv.mkDerivation (finalAttrs: {
     homepage = "https://github.com/containers/libkrunfw";
     license = with licenses; [ lgpl2Only lgpl21Only ];
     maintainers = with maintainers; [ nickcao RossComputerGuy ];
-    platforms = [ "x86_64-linux" "aarch64-linux" ];
+    platforms = [ "x86_64-linux" ] ++ lib.optionals (!sevVariant) [ "aarch64-linux" ];
   };
 })

